examples/nidaqmx/analog-input-everynsamples.py

```diff
@@ -0,0 +1,132 @@
+# Demonstrates how to acquire analog data using the grpc asyncio API and RegisterEveryNSamplesEvent.
+#
+# The gRPC API is built from the C API. NI-DAQmx documentation is installed with the driver at:
+# C:\Program Files (x86)\National Instruments\NI-DAQ\docs\cdaqmx.chm
+#
+# Getting Started:
+#
+# To run this example, install "NI-DAQmx Driver" on the server machine.
+# Link: https://www.ni.com/en-us/support/downloads/drivers/download.ni-daqmx.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Running from command line:
+#
+# Server machine's IP address, port number, and physical channel name can be passed as separate command line arguments.
+#   > python analog-input.py <server_address> <port_number> <physical_channel_name>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763", with "Dev1/ai0" as the physical channel name
+import asyncio
+import grpc
+import sys
+
+import nidaqmx_pb2 as nidaqmx_types
+import nidaqmx_pb2_grpc as grpc_nidaqmx
+
+server_address = "localhost"
+server_port = "31763"
+physical_channel = "Dev1/ai0"
+
+if len(sys.argv) >= 2:
+    server_address = sys.argv[1]
+if len(sys.argv) >= 3:
+    server_port = sys.argv[2]
+if len(sys.argv) >= 4:
+    physical_channel = sys.argv[3]
+
+
+async def main():
+    client = None
+    task = None
+
+    async with grpc.aio.insecure_channel(f"{server_address}:{server_port}") as channel:
+        try:
+            client = grpc_nidaqmx.NiDAQmxStub(channel)
+
+            async def raise_if_error(response):
+                if response.status:
+                    response = await client.GetErrorString(
+                        nidaqmx_types.GetErrorStringRequest(
+                            error_code=response.status, buffer_size=2048))
+                    error_string = response.error_string.strip(" \0")
+                    raise Exception(f"Error: {error_string}")
+
+            async def raise_if_error_async(awaitable_call):
+                response = await awaitable_call
+                await raise_if_error(response)
+                return response
+
+            create_response: nidaqmx_types.CreateTaskResponse = (
+                await raise_if_error_async(
+                    client.CreateTask(nidaqmx_types.CreateTaskRequest())
+                )
+            )
+            task = create_response.task
+
+            await raise_if_error_async(
+                client.CreateAIVoltageChan(
+                    nidaqmx_types.CreateAIVoltageChanRequest(
+                        task=task,
+                        physical_channel=physical_channel,
+                        min_val=1.0,
```

Comment:
Did you copy this from another example? I thought the standard range was usually 0-10 V, although it's not a big deal...

Reply:
Oops. The other ones are (-10,10). I updated to that. Anyway it should include zero.